med_io/pipeline.py

```diff
@@ -89,14 +90,19 @@ def _map(*args):
 
             patchs_imgs, patchs_labels, index_list = get_patches_data(max_data_size, patch_size, images_data,
                                                                       labels_data,
-                                                                      patches_indices, slice_channel_img=input_slice,
+                                                                      patches_indices,
+                                                                      slice_channel_img=input_slice,
                                                                       slice_channel_label=output_slice,
                                                                       output_patch_size=config['model_output_size'],
+                                                                      random_shift_patch=config['random_shift_patch'],
```

Comment:
add to config check

Reply:
The parameter random_shift_patch already exists in the default config, is a config check necessary in that case?
If yes I would suggest using random_shift_patch=True as standard, because this is also the value in the default config and the default value of the get_patches_data function

Reply:
sorry, my oversight. This is not needed then.